db/migrate/20230515074111_add_employee_id_to_vacations.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmployeeIdToVacations < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :vacations, :employee, null: false, foreign_key: true
```

Comment:
no need for additional migration, you could have added it in this file[db/migrate/20230511171610_create_vacations.rb]

Reply:
Remove this file